src/index.js

```diff
@@ -1,26 +1,22 @@
-import 'react-app-polyfill/ie11'; // For IE 11 support
-import 'react-app-polyfill/stable';
-import './polyfill'
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import { Provider } from "react-redux";
+import { createStore, combineReducers, compose, applyMiddleware } from "redux";
+import thunk from "redux-thunk";
+import { authReducer } from "./reducers/authReducer";
 
-import { icons } from './assets/icons'
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
-import { Provider } from 'react-redux'
-import store from './store'
+const rootReducer = combineReducers({
+  auth: authReducer
+});
+const store = createStore(rootReducer, composeEnhancers(applyMiddleware(thunk)));
```

Comment:
Why are we creating a store again? There is already one present imported from `./store`